### ISSUE TITLE
Community - Download assets tab missing

### ISSUE THREAD

**Issue by rwbaer** [makehumancommunity/makehuman#207]
Nightly build windows: 20220719 alpha (HEAD:f8ff3bc5)
As the issue title says the `download` assets tab is completely gone.  Checking settings shows all boxes checked.

`
Could not load 8_asset_downloader
Traceback (most recent call last):
  File "./core\mhmain.py", line 585, in loadPlugin
    module.load(self)
  File "C:\Program Files\makehuman-community\makehuman\plugins\8_asset_downloader\__init__.py", line 38, in load
    downloadView = category.addTask(AssetDownloadTaskView(category))
  File "C:\Program Files\makehuman-community\makehuman\plugins\8_asset_downloader\assetdownload.py", line 72, in __init__
    self.assetdb = AssetDB(self)
  File "C:\Program Files\makehuman-community\makehuman\plugins\8_asset_downloader\assetdb.py", line 67, in __init__
    self._loadRemoteDB()
  File "C:\Program Files\makehuman-community\makehuman\plugins\8_asset_downloader\assetdb.py", line 92, in _loadRemoteDB
    self.remoteJson = json.load(f)
  File "D:\obj\windows-release\37amd64_Release\msi_python\zip_amd64\__init__.py", line 296, in load
  File "D:\obj\windows-release\37amd64_Release\msi_python\zip_amd64\__init__.py", line 348, in loads
  File "D:\obj\windows-release\37amd64_Release\msi_python\zip_amd64\decoder.py", line 337, in decode
  File "D:\obj\windows-release\37amd64_Release\msi_python\zip_amd64\decoder.py", line 355, in raw_decode
json.decoder.JSONDecodeError: Expecting value: line 1 column 1 (char 0)

`

**Comment by pabu** [comment]
<img width="1920" alt="Download_assets_tab" src="https://user-images.githubusercontent.com/2013631/179870183-38d7250d-8bbc-4a78-b2c3-4f53d32f05b1.png">

If the tab in the screenshot is what you are looking for, then It shows for me both when running nightly installer and when cloning git repo.

**Comment by rwbaer** [comment]
Yes.  That's the tab.  Can you confirm that your version is Nightly build windows: 20220719 alpha (HEAD:f8ff3bc5)?

Are you on Windows 11?

**Comment by rwbaer** [comment]
Okay.  I just tried again on my home computer and the Download assets tab shows up fine.  Must have been a bad install somehow.  Closing this issue.

**Comment by rwbaer** [comment]
20220720 alpha (HEAD:f8ff3bc5)
I'm reopening.  

On the same work computer that failed yesterday, the community tabe is still missing.  I tried first to reunzip yesterdays and resintsall.  Fail to see tab.  I then downloaded a fresh nightly today (but apparently the same build) and got no tab again.

Like pabu, the install worked fine for one of my home computers.  I try again on yet another work computer later.

**Comment by rwbaer** [comment]
So, I think pabu is essentially right (although I used a hammer instead of finesse).  I deleted my "C:\Users\%USERNAME%"\Documents\makehuman" folder and upon restarting, MH noticed the missing assetDB and created everything again.  THe community downloadassets tab is back.  It seems unfortunate that corruption of the database is not handled a little more gently, but since this is the only time I've seen it starting over is not a disaster.

Closing the issue a second time.
